src/styles/_fonts.scss

```diff
@@ -0,0 +1,42 @@
+@font-face {
```

Comment:
Why not just import the font from [Google Fonts](https://fonts.google.com/specimen/Saira?query=saira) by placing the `<link>` tag in the `public/index.html` file?

Reply:
yes, it could be just the <link> instead of downloading fonts from Google, this approach would be easier. Should I change it? I did it out of habbit :)